website/docs/installation/documentdb/deb.md

```diff
@@ -28,9 +28,21 @@ Ensure to replace `/path/to/documentdb.deb` with the actual path and filename of
 Once installed, update your `postgresql.conf` to load the extension libraries on startup into the default `postgres` database.
 Add the following lines to `postgresql.conf`:
 
+<!-- Keep in sync with https://github.com/FerretDB/documentdb/blob/ferretdb/ferretdb_packaging/10-preload.sh -->
+
 ```text
-shared_preload_libraries = 'pg_cron,pg_documentdb_core,pg_documentdb'
-cron.database_name       = 'postgres'
+shared_preload_libraries                      = 'pg_cron,pg_documentdb_core,pg_documentdb'
+cron.database_name                            = 'postgres'
+
+documentdb.enableLetAndCollationForQueryMatch = true
```

Comment:
Is there any known limitation for not adding the others or should this be the standard going forward? The recent CNPG blog post I did only had the first 2

Reply:
What are the others are you talking about?

We were requested schema validation recently, so we enabled `documentdb.enableSchemaValidation and we also decided to enable more features.

Reply:
We already had the ones below and that's what I wrote in the CNPG blog post  so wanted to know if the new additions are mere cosmetic features or actual necessities.

```
shared_preload_libraries                      = 'pg_cron,pg_documentdb_core,pg_documentdb'
cron.database_name                            = 'postgres'
```

Reply:
Ah I see :hugs:, so changes in `documentdb.enableUserCrud` and `documentdb.maxUserLimit` are cosmetic, as it was set in the code, see the change https://github.com/FerretDB/FerretDB/pull/5086/files#diff-15905a12ca8d7222ae023b855763214684cc6a76a78176c2b45ba7f2e80bb3e8L146-L153

The new features that enabled are 
```
documentdb.enableLetAndCollationForQueryMatch = true
documentdb.enableNowSystemVariable            = true
documentdb.enableSortbyIdPushDownToPrimaryKey = true

documentdb.enableSchemaValidation             = true
documentdb.enableBypassDocumentValidation     = true
```

Reply:
Thanks for this explanation 🤗